cmd/opm/serve/serve.go

```diff
@@ -59,12 +68,32 @@ will not be reflected in the served content.
 	}
 
 	cmd.Flags().BoolVar(&s.debug, "debug", false, "enable debug logging")
+	cmd.Flags().StringVar(&s.profilePort, "profileport", "", "pprof HTTP endpoint port")
```

Comment:
My personal opinion is that we could have a high-level `enable-pprof` flag and then some sane defaults for the  flags, WDYT? Having to specify specific ports and files might be annoying if a user just wants to see some data.

Reply:
I went both ways on this, because the write paths could be vastly different between the pod and local executions of this tool (plus some unanticipated runtime environment).  
At the end, the choices came down to versatility (and vetting the data for security/access/etc) or using some well-communicated defaults which should hit the majority of cases.  

I landed with high-level file/endpoint enabling flags, and sane defaults. :)